### ISSUE TITLE
Use VPN only for internal resources is ignoring VPN's DNS server and automatic routes

### ISSUE THREAD

**Issue by felipeaf** [enaess/network-manager-sstp#37]
I had installed eoan packages from ppa. I could connect to my company VPN with that, but the option to not redirect all my Internet traffic to VPN is not working.
The "Use this connection only for resources on this network"* checkbox in VPN settings (gnome settings, network, VPN) doesn't works correctly. When that checkbox is marked:
1 - It's not using VPN's DNS, even when i try configure that manually in gnome with that server address, whether is Automatic DNS on or OFF (so i cannot resolve domain address from VPN network)
2- It only works when manually defining routes for the private ip ranges ("automatic" should works like in MS Windows, i guess it should try use VPN to reach ips in private ip range, like 10.x.x.x or 192.168.x.x).

$lsb_release -rd
Description:	Ubuntu 19.10
Release:	19.10

$ apt-cache policy network-manager-sstp-gnome
network-manager-sstp-gnome:
  Instalado: 1.2.6-0ubuntu3
  Candidato: 1.2.6-0ubuntu3
  Tabela de versão:
 *** 1.2.6-0ubuntu3 500
        500 http://ppa.launchpad.net/eivnaes/network-manager-sstp/ubuntu eoan/main amd64 Packages
        100 /var/lib/dpkg/status

$ apt-cache policy network-manager-sstp
network-manager-sstp:
  Instalado: 1.2.6-0ubuntu3
  Candidato: 1.2.6-0ubuntu3
  Tabela de versão:
 *** 1.2.6-0ubuntu3 500

**Comment by enaess** [comment]
I don't think you are the only one reporting this. I'll need to take a look, but been kind of hammered with things at work. Maybe get to it this weekend or next. Also, I've been trying to get another release out with certificate support such that users can use this against Microsoft Azure.
